markdown/report.Rmd

```diff
@@ -166,121 +241,164 @@ More recently in December 2016 the ["Health and Social Care Delivery Plan"](http
 
 ### Provisional Figures 
 
-Note that hospital data and National Records of Scotland deaths data are provisional for 2018/19. This publication will be updated in October 2019 with final figures.  Figures for the latest year should therefore be treated with caution as there could be an undercount of hospital admissions, particularly for people who died in the last three months of the year.  This is not expected to have a large impact on these published figures when they are revised in subsequent updates. See [**Appendix 2**](#Appendix2) for more information.
+Note that hospital data and National Records of Scotland deaths data are provisional for `r latest$fy`. This publication will be updated in October `r year(end_date)` with final figures.  Figures for the latest year should therefore be treated with caution as there could be an undercount of hospital admissions, particularly for people who died in the last three months of the year.  This is not expected to have a large impact on these published figures when they are revised in subsequent updates. See [**Appendix 2**](#Appendix2) for more information.
 
 
 <Div custom-style = "Heading 2_pgbrk">Main Points</div>
 
-*	In 2018/19, there were 53,168 deaths in Scotland excluding those where an external cause such as unintentional injury was recorded. For these individuals, 89.2% of their last six months of life was spent either at home or in a community setting, with the remaining 10.8% spent in hospital. This is equivalent to an average of 20 days in hospital in the six months prior to death.
+* In `r latest$fy`, there were `r latest$deaths` deaths in Scotland excluding those where an external cause of death was recorded (including falls).  For these individuals, `r latest$qom`% of their last six months of life was spent either at home or in a community setting, with the remaining `r latest$qom_hosp`% spent in hospital. This is equivalent to an average of `r round_half_up(182.5 * (latest$qom_hosp / 100))` days in hospital in the six months prior to death.
+
+* Over the past `r as.character(as.english(year(end_date) - year(start_date)))` years, the percentage of time spent at home or in a community setting has gradually increased from `r summarise_data(basefile, include_years = "first") %>% pull(qom)`% in `r min(basefile$fy)` to `r latest$qom`% in `r latest$fy`.
 
-*	Over the past nine years, the percentage of time spent at home or in a community setting has gradually increased from 85.3% in 2010/11 to 89.2% in 2018/19.
+* In `r latest$fy`, `r max_old`s aged 85 and older spent a higher percentage of their last six months of life in the community (`r max(old$qom)`%) compared to `r min_old`s (`r min(old$qom)`%); this equates to `r max_old`s spending, on average an extra `r as.character(day_diff(max(old$qom), min(old$qom)))` days in the community in their last six months of life.
```

Comment:
I see in your report it says "aged 85 and over". Do you want to change it to "older"? Doesn't make a difference but want to double check.

Reply:
Thanks but want to keep as over

Reply:
Okay then you need to change the current text to "aged 85 and over" as it's currently "older". Sorry I mean I had a look at your previous report and found it should be "over"!

Reply:
Ahhh, thanks!